docs/content/design/broker.md

```diff
@@ -63,3 +63,12 @@ Returns the metrics of the datasource.
 
 Returns a flag indicating if the broker knows about all segments in Zookeeper. This can be used to know when a broker node is ready to be queried after a restart.
 
+* `/druid/v2/candidates/?datasource=<datasource-name>&intervals=<comma-separated-intervals-in-ISO8601-format`
```

Comment:
Our APIs are not very consistent between the coordinator and the broker here
Coordinator returns almost identical information at `/druid/coordinator/v1/datasources/{dataSourceName}/intervals/{interval}/serverview` but uses a very different API